### PR TITLE
Update bot.dart

### PR DESCRIPTION
**Pull request by Daimonil**
Added a new command that sends real music

### CODE REVIEW

**Review by dmbaranov — CHANGES_REQUESTED** [review]
Remove the wrong log, please.

File: lib/src/bot.dart [hunk 1]
```diff
@@ -325,4 +326,22 @@ class Bot {
...
+    if (message.text == null) {
+      await message.reply('Нахуй пошол, мудило!!1');
+      return;
+    }
+
+    var rawText = message.text.split(' ');
+    var text = rawText.sublist(1).join(' ');
+    text = text.replaceAll ('music.', '');
+
+    print('${message.from.toJson()} is writing to Coop: ${message.toJson()}');
```
[the remark this answers is not in the document]
Comment (Daimonil) [reply lib/src/bot.dart:339]: Yep, just privat bot command

File: lib/src/bot.dart [hunk 2]
```diff
@@ -325,4 +326,22 @@ class Bot {
...
     await telegram.sendMessage(chatId, joke.joke);
   }
-}
+  void _sendRealMusic(TeleDartMessage message) async {
+    if (message.text == null) {
+      await message.reply('Нахуй пошол, мудило!!1');
+      return;
+    }
+
+    var rawText = message.text.split(' ');
```
Comment (dmbaranov) [inline lib/src/bot.dart:335]: I also suggest adding a check that the link that the user has sent is indeed leads to YouTube music. Something like .startsWith('music.youtube.com')

**Review by dmbaranov — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**